styles/mixins/triangle.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = function( config ) {
```

Comment:
Its an icon, so it seems like a perfect use case.  Requires 1 element, scales with the font, changes color with the parent, easier to style (doesn't require 2 selectors).